helix-sitemap.yaml

```diff
@@ -0,0 +1,81 @@
+version: 1
+
+sitemaps:
+  website:
+    origin: https://www.adobe.com
+    languages:
+      default:
+        source: /express/query-index.json
+        destination: /express/sitemap-en.xml
```

Comment:
`/express/sitemap-us.xml` is the old location, didn't want to overwrite it until its good